### ISSUE TITLE
How to stop logging and view error report before sending

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-documentation#612]
## Avoid duplicates
* [X] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
https://www.coronawarn.app/en/faq/#error_log and
https://www.coronawarn.app/en/blog/2021-05-12-corona-warn-app-version-2-2/
do not answer the question.
* [X] Affects Android and iOS
* [X] Question has not already been asked in another issue

## Your Question

The question is how to view exactly the data which the Error Report sends to technical support. There seems to be no way to stop error logging, view the data, then send it.

Also I could find no way, after the data has been sent, to view what data was sent. The only remnant on the mobile device seems to be the timestamp and the ID.

Issues first raised in App version: 2.2.1
Reviewed for consistency with App version: 2.17.2

In App Information > Error Reports, the text (from string `debug_log_privacy_card_container_section_two`) says:

"Initially, only you will have access to the error report. You can then decide whether to send the error report to the technical support team, whether to save the error report on your smartphone first, or whether to stop and erase the recorded information. **If you first save the error report locally, this will give you the opportunity to view the error report yourself before submitting it to support.**"

![image](https://user-images.githubusercontent.com/66998419/118269316-e1340200-b4be-11eb-9d6e-206d04e89e65.png)

After tapping "START" there is no option to STOP and VIEW the data. 
- The "STOP AND DELETE" button does what it says, so there is no data to view.
- The "SHARE AND CONTINUE" button opens up the Android Share option (by default with apps Drive, Bluetooth and Gmail available) however "Recording Active" is still shown, so it is still collecting data as can also be seen by the fact that the Current size continues to increase.
- The "SEND ERROR REPORT" only provides the option to "AGREE AND SEND".

![image](https://user-images.githubusercontent.com/66998419/118269920-95358d00-b4bf-11eb-90ec-b7ca96cd8265.png)

---
Internal Tracking ID: [EXPOSUREAPP-7292](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7292)

**Comment by MikeMcC399** [comment]
@dsarkar 
I agree with your assessment. Let me know your recommendation about how to proceed now.

**Comment by dsarkar** [comment]
@MikeMcC399 

- There is no way to:  STOP -> (optionally) VIEW -> (optionally) SEND the error log
- Consequently, there here is no way to see the FULL report before sending
- In that case the describing txt is a bit mis-leading
- it is not obvious/intuitive than one can view the ongoing error report via  SHARE AND CONTINUE button 

Suggestion to suggest internally these options to fix this issue:

- Adapt the text to the "reality" and maybe explain the SHARE button better and/or rename the SHARE button. 
- Or to change the work flow:
  - Introduce separate functions/buttons/menu: START, STOP, CONTINUE, DELETE error logging
  - Introduce separate functions/buttons/menu: SEND to technical support, SHARE/VIEW error report

@MikeMcC399 anything you would like to add/change? Thanks.

**Comment by MikeMcC399** [comment]
@dsarkar 
Thanks for the summary! I will submit a separate issue with repro steps to keep it simpler, and I will link to this issue.

**Comment by MikeMcC399** [comment]
@dsarkar 
I have created the issue "Cannot view exact error report before submitting" #614. I think the function STOP (without DELETE) is missing. Let's wait for dev feedback now. Your suggestions about modifying the workflow of course are also valid.

I also created the issue "Timestamp of shared error report uses UTC" https://github.com/corona-warn-app/cwa-app-android/issues/3183 which I noticed whilst I was capturing the information about the first issue.
